src/communication.ts

```diff
@@ -12,55 +15,47 @@ export class Communication {
         });
     }
 
-    /**
-     * connect unregistered user to the website
-     * @param randomSeed the random seed the user need to connect on typically recieved in jwt
+    /* connects to the Tonomy Communication server, authenticates with it's DID
+     * subscribes to any messages that are sent by `sendMessage` by providing a callback function executed every time a message is received
+     * should send a read receipt when messages are received
+     * @returns true if successful
      */
-    connectTonomy(randomSeed: string) {
-        this.socketServer.emit('connectTonomy', { randomSeed });
-    }
+    login(authorization: Message): Promise<boolean> {
+        return new Promise((resolve, reject) => {
```

Comment:
probably should do a check that it is an authorization message for login
we could create a subclass `LoginMessage` for this. New Task